eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/SiteAccessAware/Contextualizer.php

```diff
@@ -73,6 +78,23 @@ public function mapConfigArray($id, array $config, $options = 0)
             []
         );
 
+        // (!) Keep SiteAccess group settings
+        foreach (array_keys($this->availableSiteAccessGroups) as $scope) {
+            $scopeSettings = $config[$this->siteAccessNodeName][$scope][$id] ?? [];
+            if (empty($scopeSettings)) {
+                continue;
+            }
+
+            $mergedSettings = $this->mergeSettingsForSiteAccessGroup(
+                $options,
+                $defaultSettings,
+                $scopeSettings,
+                $globalSettings
+            );
+
+            $this->container->setParameter("$this->namespace.$scope.$id", $mergedSettings);
```

Comment:
Is this interpolation needed here? 
```suggestion
            $this->container->setParameter($this->namespace.$scope.$id, $mergedSettings);
```